akka-persistence-testkit/src/main/scala/akka/persistence/testkit/internal/EventSourcedBehaviorTestKitImpl.scala

```diff
@@ -175,7 +175,7 @@ import akka.stream.scaladsl.Sink
     if (serializationSettings.enabled && serializationSettings.verifyCommands)
       verifySerializationAndThrow(command, "Command")
 
-    if (serializationSettings.enabled && !emptyStateVerified) {
+    if (serializationSettings.verifyState && !emptyStateVerified) {
```

Comment:
shouldn't this be
```
if (serializationSettings.enabled && serializationSettings.verifyState && !emptyStateVerified) {
```
?

That is how it is used in other places, such as `postCommandCheck`